validation/static/index.js

```diff
@@ -155,5 +171,24 @@ document.addEventListener('DOMContentLoaded', () => {
             })
         }
     }
-
-})
\ No newline at end of file
+})
+
+function showWrongWordDiv(recordingId) {
+    const wrongWordDivs = document.getElementsByClassName("rec-wrong-word");
+    for (let d of wrongWordDivs) {
+        if (d.dataset.recId === recordingId) {
+            d.classList.remove('menu__none')
+            d.classList.add('menu__block')
+        }
+    }
+}
+
+function hideWrongWordDiv(recordingId) {
+    const wrongWordDivs = document.getElementsByClassName("rec-wrong-word");
+    for (let d of wrongWordDivs) {
+        if (d.dataset.recId === recordingId) {
+            d.classList.add('menu__none')
+            d.classList.remove('menu__block')
+        }
+    }
+}
```

Comment:
I'm noticing that this pattern continues to be duplicated:

```javascript
const elements = document.getElementsByClassName(SOME_CLASS_NAME);
for (let e of elements) {
    if (e.dataset.recId === recordingID) {
      e.classList.add(OLD_CLASS);
      e.classList.remove(NEW_CLASS);
    }
}
```

Since there's two things going on here, I suggest two Extract Function refactorings:

```javascript
function* yieldElementWithClassAndRecordingId(className, recordingId, calback) {
  const elements = document.getElementsByClassName(SOME_CLASS_NAME);
  for (let e of elements) {
      if (e.dataset.recId === recordingID) {
        yield e
      }
  }
}
```

And you can use `.replace()` instead of `.add()` and `.remove()`:

```javascript
e.classList.replace(OLD_CLASS_NAME, NEW_CLASS_NAME)
```

This will result in code like:

```suggestion
function showWrongWordDiv(recordingId) {
    for (let d of yieldElementWithClassAndRecordingId("rec-wrong-word", recordingID)) {
        d.classList.replace('menu__none', 'menu__block')
    })
}

function hideWrongWordDiv(recordingId) {
    for (let d of yieldElementWithClassAndRecordingId("rec-wrong-word", recordingID)) {
        div.classList.replace('menu__block', 'menu__none')
    })
}
```

Reply:
`classList.replace` works on everything except the header. The header is in 1 or 3 states and that state is unknown. If I try to replace a class that isn't being applied to the header, it fails and the header winds up with no style. Otherwise this refactor is awesome!